happypose_ros/happypose_ros/happypose_node.py

```diff
@@ -631,6 +642,26 @@ def _await_results(self, cam_data: dict) -> None:
                     results, header, has_bbox=not self._multiview
                 )
 
+                ##################
+                # for contact_graspnet_ros
+                masks = results["masks"].numpy()
+                merged_mask = np.zeros(masks[0].shape, dtype=np.uint16)
+                for instance_id in range(len(masks)):
+                    # label format: tless-obj_000021
+                    label = results["infos"].iloc[instance_id].label
```

Comment:
This is expected to work only for tless?
Maybe an assert ?
```suggestion
                    # label format: tless-obj_000021
                    label = results["infos"].iloc[instance_id].label
                    assert label.startswith("tless-obj_")
```

Reply:
All 2D detected object labels follow the same f"<dataset>-obj_{obj_id:06}" pattern, so this should work for all datasets. I can make it more clear in the comment.

Reply:
Noted. Thanks.